### PR TITLE
Feature/apicalls

### PR DESCRIPTION
**Pull request by asiisii**
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant screenshots of your code being deployed. 
- add methods to make api calls
- fetched data and passed down to home component
- displayed each pokemon as a card with pokemoncard component

Fixes # (issue)
- #2 

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

## Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/apiData/apiCalls.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+const baseURL = 'https://pokeapi.co/api/v2/'
+const getAll = 'https://pokeapi.co/api/v2/pokemon?limit=151'
+const singlePokemon = 'https://pokeapi.co/api/v2/pokemon/1'
+const soloPic = 'https://pokeres.bastionbot.org/images/pokemon/1.png'
+
+const apiCalls = async () => {
```
Comment (Thee-Dust) [inline src/apiData/apiCalls.js:7]: nice use of async and await

Comment (rhen92) [inline src/apiData/apiCalls.js:7]: Nice use of async and await! Teaching me new things!

Comment (asiisii) [reply src/apiData/apiCalls.js:7]: thank you

**Review by Thee-Dust — COMMENTED** [review]
Looking good!

File: src/components/PokemonCard/PokemonCard.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const PokemonCard = ({id, img, name}) => {
+  return (
+    <>
```
Comment (rhen92) [inline src/components/PokemonCard/PokemonCard.js:5]: Didn't know you could use empty tags interesting how it still works.

**Pull request merged**